### DISCUSSION TITLE
calc 2.14.0.9 release

### DISCUSSION THREAD

**Discussion by lcn2** [lcn2/calc#39] [Announcements]
The following are the changes in this release:

    Due to issues with clang and Apple Silicon, ARCH_CFLAGS is now,
    by default, empty:

	ARCH_CFLAGS=

    If you want to use, say, -march=native, then either change
    the Makefile or make with:

	make all ARCH_CFLAGS='-march=native'

    Added arch and hardware as GNU Makefile computed values.
    As with ${target}, ${arch} and ${hardware} is computed by uname:

	target: uname -a
	arch: uname -p
	hardware: uname -m

    Fixed compiling calc on Apple Silicon with homebrew.  On Apple
    Silicon, HomeBrew installs on into a different location.  The
    Makefile checks if ${hardware} is arm64 and adjusts the location
    of libraries such as readline and history accordingly.

<hr /><em>This discussion was created from the release <a href='https://github.com/lcn2/calc/releases/tag/v2.14.0.9'>calc 2.14.0.9 release</a>.</em>

**Comment by pmetzger** [comment]
FWIW, MacPorts has had `ARCH_CFLAGS:=` in Makefile.local for a while.

**Comment by lcn2** [comment]
Thanks for that info about MacPorts Makefile.local @pmetzger
